apps/condo/domains/billing/schema/BillingCategory.js

```diff
@@ -1,24 +1,39 @@
 /**
  * Generated by `createschema billing.BillingCategory 'name:Text'`
  */
+const Ajv = require('ajv')
 
-const { LocalizedText } = require('@open-condo/keystone/fields')
+const { LocalizedText, Json } = require('@open-condo/keystone/fields')
 const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
 const { GQLListSchema } = require('@open-condo/keystone/schema')
 
 const access = require('@condo/domains/billing/access/BillingCategory')
+const { getValidator } = require('@condo/domains/common/schema/json.utils')
 
-
+const ajv = new Ajv()
 
 const BillingCategory = new GQLListSchema('BillingCategory', {
-    schemaDoc: 'Payment category - used primarily in display purposes',
+    schemaDoc: 'Receipt category - used primarily in display purposes',
     fields: {
         name: {
             schemaDoc: 'Localized name of billing category: Hot water, Cold water, Housing Services',
             type: LocalizedText,
             isRequired: true,
             template: 'billing.category.*.name',
         },
+        serviceNames: {
+            schemaDoc: 'Possible names of services to auto-detect receipt category',
+            type: Json,
+            isRequired: false,
+            hooks: {
```

Comment:
Is it possible to provide gql return type?

Reply:
This field will be used only in unified mutation. It only contains an array of strings and I didn't include this field to gql queries